kolibri/plugins/document_pdf_render/assets/src/vue/index.vue

```diff
@@ -117,6 +117,8 @@
     &:fullscreen
       width: 100%
       height: 100%
+      min-height: inherit
+      max-height: inherit
```

Comment:
just wanted to make sure that this change (in the `document_pdf_render` plugin) was intended for this PR?